test/llilc_runtest.py

```diff
@@ -171,7 +173,8 @@ def main(argv):
     # Run the test
     return_code = const.RunTestOK
     exclusion = os.path.join(os.path.dirname(__file__), 'exclusion.targets')
-    runtest_command = 'runtest ' + args.arch + ' ' + args.build 
+    runtest_full_path = os.path.join(runtest_dir, 'runtest')
```

Comment:
I'd change `'runtest'` to `'runtest.cmd`', since that's what we're testing for the existence of above.